Add foundry multisig guide (#305)
rsksmart/devportal: tutorials/ethereum-devs/setup-truffle-oz.md

### Visual Studio Code (VS Code)

> In this tutorial, we would use VS Code to create and deploy our project.

Reviewer: ```suggestion
In this tutorial, we will use VS Code to create our project.
```

Note that the code editor is not used to deploy the project -0 that is accomp0lished through another tool - Truffle in this case.

![myproject folder](/assets/img/tutorials/setup-truffle-oz/image-13.png)

> In your project folder, start an Truffle project by typing the below command into the terminal:

Reviewer: ```suggestion
In your project folder, start an Truffle project by typing the command below into the terminal:
```

There are a few ways to do this.

One is to create using a web wallet, such as [Metamask](https://metamask.io/) or [Nifty](https://www.poa.network/for-users/nifty-wallet) wallet. 
These wallets generate the mnemonic for you. 
If you wanted to create using Metamask, you can get the instructions here:

- [using Remix and Metamask with RSK testnet](/tutorials/ethereum-devs/remix-and-metamask-with-rsk-testnet/)

## iancoleman.io/bip39

> 

Reviewer: Not related to this PR: Now that
[iancoleman.io/bip39 has support for RSK](https://github.com/iancoleman/bip39/pull/409),
we should include screenshots of that in this section.
